runtime/resolvers/metrics_sql.go

```diff
@@ -16,7 +17,8 @@ func init() {
 }
 
 type metricsSQLProps struct {
-	SQL string `mapstructure:"sql"`
+	SQL             string `mapstructure:"sql"`              // SQL is the metrics SQL to evaluate.
+	AdditionalWhere string `mapstructure:"additional_where"` // AdditionalWhere is a filter to apply to the metrics SQL. (additional WHERE clause)
```

Comment:
The frontend will want to provide this as a `metricsview.Expression` type, not a `string` (that's why they need this and can't just put it in the metrics SQL).

I think you can just use the `metricsview.Expression` type directly here, it's serializable/deserializable.